README.md

```diff
@@ -278,10 +278,10 @@ To generate the predictions on the Semantic KITTI validation set, type:
 $ cd MonoScene/
 $ python monoscene/scripts/generate_output.py \
     +output_path=$MONOSCENE_OUTPUT \
-    dataset=kitti \
-    kitti_root=$KITTI_ROOT \
-    kitti_preprocess_root=$KITTI_PREPROCESS \
-    n_gpus=1 batch_size=1
+    +dataset=kitti \
+    +kitti_root=$KITTI_ROOT \
+    +kitti_preprocess_root=$KITTI_PREPROCESS \
+    +n_gpus=1 +batch_size=1
```

Comment:
Are you sure we need to add "+" ? As these parameters exist in the [config file,](https://github.com/astra-vision/MonoScene/blob/master/monoscene/config/monoscene.yaml#L7) I don't think we need to have "+" before.